### DISCUSSION TITLE
localization failure when use Nuitka or Pyinstaller compile python script

### DISCUSSION THREAD

**Discussion by funchan24** [israel-dryer/ttkbootstrap#306] [General]
OS: Windows 11 X64
Pyhong: 3.7.6 32BIT
ttkbootstrap: 1.8.0
Nuitka: 0.9.2
Pyinstaller: 5.1

run at windows server 2008r2
before compile
<img width="404" alt="656d11d489c71802ee29c3a29df5bb8" src="https://user-images.githubusercontent.com/42088233/176990327-ee731e10-9468-4715-a311-e6d0db4fc2ba.png">

after compile
<img width="404" alt="3c798aa2b8bf598d444bb21825e51da" src="https://user-images.githubusercontent.com/42088233/176990350-0907c6a9-bc09-4c26-8264-a22d131a4dfa.png">

**Comment by funchan24** [comment]
@israel-dryer

**Comment by TheDemonGuard** [comment]
@funchan24 
I can also reproduce this behavior.
But not sure whether this behavior is specific to ttkbootstrap or is a general issue.

> OS: Windows 11 X64 Pyhong: 3.7.6 32BIT ttkbootstrap: 1.8.0 Nuitka: 0.9.2 Pyinstaller: 5.1
> 
> run at windows server 2008r2 before compile <img alt="656d11d489c71802ee29c3a29df5bb8" width="404" src="https://user-images.githubusercontent.com/42088233/176990327-ee731e10-9468-4715-a311-e6d0db4fc2ba.png">
> 
> after compile <img alt="3c798aa2b8bf598d444bb21825e51da" width="404" src="https://user-images.githubusercontent.com/42088233/176990350-0907c6a9-bc09-4c26-8264-a22d131a4dfa.png">
